doc/nrf/known_issues.rst

```diff
@@ -1702,7 +1740,22 @@ NCSDK-7982: Partition manager: Incorrect partition size linkage from name confli
 DFU and FOTA
 ============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-1-2 v2-1-1 v2-1-0
+
+NCSDK-18357: Serial recovery does not work on nRF5340 network core
+  If a network core serial recovery is attempted using MCUboot serial recovery, the upload will complete, but the image will not be transferred to the network core and the upgrade will fail.
+
+.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+NCSDK-18422: Serial recovery fails to write to slots in QSPI
+  If a slot resides in an external QSPI storage area and this area is written to in MCUboot's serial recovery system, the writing will not work due to memory buffer alignment offsets and requirements with the underlying flash driver.
+
+.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSDK-18108: ``s1`` variant image configuration mismatch
+  If an image with an ``s1`` variant is configured and the ``s0`` image configuration is changed using menuconfig, these changes will not be reflected in the ``s1`` configuration, which can lead to a differing build configuration or the build does not upgrade.
```

Comment:
Hi @tejlmand,
Do you think we can link to cherry-picking `e4be060f8b00bc386dbf9cc133f194bf8a1ccdc2` as a workaround here?

Reply:
Checked and this one does apply cleanly to v2.1.2 so it can stay

Reply:
but what about `v2.0.x` and `v1.9.x` ?
This is marked as a know issue on all those releases.

Remember, know issues are not tagged per release but is dynamic and it's now presented as applicable on v1.9.x:
![image](https://user-images.githubusercontent.com/9802583/206148140-6d5d32b2-a222-4597-ba9c-2663b5234311.png)